cassdegrees/templates/globalrequirementswidget.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
This is a widget - duplicate definition of HTML might break stuff

Reply:
Thanks, wasn't sure about this one. Removed